src/widgets/SensorHeader/LastSensorDownload.js

```diff
@@ -95,7 +95,7 @@ const stateToProps = (state, props) => {
   const isDownloading = selectIsDownloading(state, macAddress);
 
   const sensor = selectSensorByMac(state, macAddress);
-  const { logDelay } = sensor;
+  const { logDelay } = sensor ?? {};
```

Comment:
🏢  🚗  🐛  office road bug?

Reply:
ah nice. I did the same in another PR too!